extensions/industry/healthcare/fhir/datatypes/availability.schema.json

```diff
@@ -0,0 +1,100 @@
+   ],
+   "$schema":"http://json-schema.org/draft-06/schema#",
+   "$id":"https://ns.fhir.org/datatypes/availability",
+   "title":"Availability",
+   "type":"object",
+   "description":"Availability data for an {item}.(This datatype is created as per HL7 FHIR Release 5 specifications)",
+   "definitions":{
+      "availability":{
+         "properties":{
+            "xdm:availableTime":{
+               "title":"Avaialble Time",
+               "description":"Times the {item} is available.",
+               "type":"array",
+               "items":{
+                  "type":"object",
+                  "additionalProperties":false,
+                  "properties":{
+                     "xdm:daysOfWeek":{
+                        "title":"Days Of Week",
+                        "description":"mon | tue | wed | thu | fri | sat | sun",
+                        "type":"array",
+                        "items":{
+                           "type":"string",
+                           "enum":[
+                              "mon",
+                              "tue",
+                              "wed",
+                              "thu",
+                              "fri",
+                              "sat",
+                              "sun"
+                           ],
+                           "meta:enum":{
+                              "mon":"Monday",
+                              "tue":"Tuesday",
+                              "wed":"Wednesday",
+                              "thu":"Thursday",
+                              "fri":"Friday",
+                              "sat":"Saturday",
+                              "sun":"Sunday"
+                           }
+                        }
+                     },
+                     "xdm:allDay":{
+                        "title":"All Day",
+                        "type":"boolean",
+                        "description":"Always available? i.e. 24 hour service."
+                     },
+                     "xdm:availableStartTime":{
+                        "title":"Available Start Time",
+                        "description":"Opening time of day (ignored if allDay = true)",
+                        "type":"string",
+                        "format":"time"
+                     },
+                     "xdm:availableEndTime":{
+                        "title":"Available End Time",
+                        "description":"Closing time of day (ignored if allDay = true)",
+                        "type":"string",
+                        "format":"time"
+                     }
+                  }
+               }
+            },
+            "xdm:notAvailableTime":{
+               "title":"NotAvailable Time",
+               "description":"Not available during this time due to provided reason..",
+               "type":"array",
+               "items":{
+                  "type":"object",
+                  "additionalProperties":false,
```

Comment:
We need to delete line "additionalProperties":false

Reply:
my bad! I should have run grep on the folder..I noticed a few more 
1. person.schema.json
2. patient.schema.json

should those be deleted too?